lib/beaker-hostgenerator/data.rb

```diff
@@ -779,6 +779,22 @@ def osinfo
             'template' => 'ubuntu-1610-x86_64'
           }
         },
+        'vro6-64' => {
+          :general => {
+            'platform' => 'sles-11-x86_64'
+          },
+          :vmpooler => {
+            'template' => 'vro-6-x86_64'
+          }
+        },
+        'vro7-64' => {
+          :general => {
+            'platform' => 'sles-11-x86_64'
```

Comment:
Thanks.

We should probably have a ticket for updating that Rakefile to use the new platform rather than append to the YAML file.  Or if possible we should just do it now since a ticket like that is easily lost in the backlog forever.